src/text/index.tsx

```diff
@@ -16,7 +16,7 @@ export enum TextDisplayType {
   TITLESTRONG = 'titleStrong',
 }
 
-export enum TextContainerType {
+export enum TextTagType {
```

Comment:
TagType might be enough

Reply:
I'll keep it that way to be consistant with the `TextDisplayType` above